node/src/BaseClient.ts

```diff
@@ -2242,6 +2244,37 @@ export class BaseClient {
         return this.createWritePromise(createLIndex(key, index));
     }
 
+    /**
+     * Inserts `element` in the list at `key` either before or after the `pivot`.
+     *
+     * See https://valkey.io/commands/linsert/ for more details.
+     *
+     * @param key - The key of the list.
+     * @param position - The relative position to insert into - either {@link InsertPosition.Before} or
+     *     {@link InsertPosition.After} the `pivot`.
```

Comment:
I don't think that javadoc links work there.

Reply:
I think JSDoc supports this, see [here](https://jsdoc.app/tags-inline-link#examples), however I don't see the link tag used elsewhere in this project. Should I remove it or keep it?

Reply:
I'm ok with `{@link ...}`, just confirm that they don't use another style for class references across client.

Reply:
There aren't many examples I could find of cross-referencing in the node client but the examples I did find use backticks instead so I'll change to that